demo/video_component/run.py

```diff
@@ -6,15 +6,25 @@
 b = os.path.join(os.path.dirname(__file__), "files/a.mp4")  # Video
 c = os.path.join(os.path.dirname(__file__), "files/b.mp4")  # Video
 
+s1 = os.path.join(os.path.dirname(__file__), "files/s1.srt")  # Subtitle
+s2 = os.path.join(os.path.dirname(__file__), "files/s2.vtt")  # Subtitle
+
+
+def video_demo(video, subtitle=None):
+    if subtitle is None:
+        return gr.update(value=video, subtitle=None)
+
+    return gr.update(value=video, subtitle=subtitle.name)
+
 
 demo = gr.Interface(
-    fn=lambda x: x,
-    inputs=gr.Video(type="file"),
+    fn=video_demo,
+    inputs=[gr.Video(type="file"), "file"],
     outputs=gr.Video(),
     examples=[
-        [a],
-        [b],
-        [c],
+        [a, s1],
+        [b, s2],
+        [c, None],
     ],
 )
```

Comment:
I wouldn't change this particular demo as it's used to illustrate the basic `Video` component in the docs. However, we could create a new `video_subtitles` demo

Reply:
Done, should I clean the subtitle files from video_componment from github history or it's ok to remain this. 
(About 600 Bytes)

Reply:
Nw about the github history, 600 bytes is fine